lib/cli/test/data/Cardano/CLISpec/key from-recovery-phrase --help.win

```diff
@@ -0,0 +1,13 @@
+Usage:  key from-recovery-phrase STYLE
+
+  Convert a recovery phrase to an extended private key
+
+Available options:
+  -h,--help                Show this help text
+  STYLE                    Byron | Icarus | Shelley | Shared
+
+The recovery phrase is read from stdin.
+
+Example:
+  [0;1m$ unit.exe recovery-phrase generate \[0m
+  [0;1m| unit.exe key from-recovery-phrase Icarus[0m
```

Comment:
Shouldn't these golden files have newlines at the end? Our CLI does print a final newline.

Reply:
I don't think they used to have: https://github.com/input-output-hk/cardano-wallet/blob/e4503c84811713cc66d857a32763e29e2994127a/lib/cli/test/unit/Cardano/CLISpec.hs#L131-L139

Perhaps the _CLI_ would _print_ an extra newline, but the string we get from optparse-applicative in `CLISpec` does not.

Reply:
OK. I like newlines at the end of text files, so I added a golden post-processing step to do that.